### DISCUSSION TITLE
Archive project

### DISCUSSION THREAD

**Discussion by beatngu13** [junit-pioneer/convert-junit4-to-junit5#59] [Announcements]
JUnit Jupiter has been available for almost 6 years now. In convert-junit4-to-junit5, there were various interactions with the community in the beginning, such as feature requests, bug reports or even pull requests. But in the last years, the project was more or less in maintenance mode. This is also due to other great projects like [OpenRewrite](https://docs.openrewrite.org), which e.g. has many different ["recipes"](https://docs.openrewrite.org/reference/recipes/java/testing/junit5) for migrating from JUnit 4 to JUnit5.

Because of this – and because we as maintainers want to focus more on JUnit Pioneer as our main project – we want to [archive](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories) convert-junit4-to-junit5. But before we do that, we would like to hear your thoughts:

* Are projects like OpenRewrite a full replacement?
* Are there important things you really want to see before archiving?
* Are you aware of any links, references, etc. that we should "fix" beforehand?
* ...

**Comment by Bukama** [comment]
I vote for archiving it.

**Comment by signed** [comment]
archiving works for me. The source code will still be around, correct?

**Reply by beatngu13** [reply]
Exactly:

>When a repository is archived, its issues, pull requests, code, labels, milestones, projects, wiki, releases, commits, tags, branches, reactions, code scanning alerts, comments and permissions become **read-only**.

https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories#about-repository-archival

**Comment by boyarsky** [comment]
As the person who started this code, I think it is awesome that this project has outlived its usefulness. It was a great bridge and I'm pleased it isn't needed anymore!

Thanks for shepherding it all this time!

**Comment by aepfli** [comment]
sidenote: i even found a fork of this repository within my companies internal VCS :D - it was definitely useful :D

**Comment by beatngu13** [comment]
Will close this discussion. Thx everyone, it was a fun ride. 🖤
